kotest-extensions/kotest-extensions-spring/src/jvmMain/kotlin/io/kotest/spring/SpringListener.kt

```diff
@@ -56,7 +59,9 @@ object SpringListener : TestListener {
          val klass = this::class.java
 
          return if (Modifier.isFinal(klass.modifiers)) {
-            println("Using SpringListener on a final class. If any Spring annotation fails to work, try making this class open.")
+            if (!ignoreSpringListenerOnFinalClassWarning || !sysprop("kotest.listener.spring.ignore.warning").toBoolean()) {
```

Comment:
Does sysprop not return a null without a default ?

Reply:
It does, but `toBoolean` is defined for nullable `String?` and it returns false.

Reply:
Ok, but it's since kotlin 1.4. I will change it to support also 1.3